STEERING-COMMITTEE.md

```diff
@@ -48,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+1. Review and approve definition of Knative Core scope.
+1. Only the Trademark Committee can redefine its own charter and authority over
+   Knative Trademark, conformance, and brand decisions.
+
+## Trademark Committee Composition
+
+The Trademark Committee is composed of company representatives from companies
+which have substantially contributed to Knative over the last year. For the 2020
+year, one seat is assigned to each of Google, IBM/RedHat, and VMware. The
+committee shall review this composition at least annually and update these
+rules.
+
+The Trademark Committee desires to always reach consensus. Additionally, each
+member can veto decisions regarding conformance and scope.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with at least half the members of the committee supporting it.
+
+Quorum is considered reached when at least of the members are present.
+
+In case of extended absence (or removal from the project by the steering
+committee), the organization of the absent member may appoint a single
+replacement delegate from the same company during the absence.
+
 ## Delegated authority
```

Comment:
I think the file now transitions from talking about trademark committee to steering committee, so we're probably missing a header somewhere

Reply:
I think this is an artifact of the haste in which this was assembled. I literally talked with Ron about it after noon Pacific yesterday, and the doc was put together sometime between 2PM (nothing) and 5:30PM (most edits in place) yesterday.

Thanks to @ronavn for turning this around so quickly!

Reply:
I agree that content dealing with a trademark committee should probably be in a distinct document for flow reasons, or at the bottom of this document.

Reply:
Done.